node-tests/acceptance/exam-test.js

```diff
@@ -11,7 +11,15 @@ function getNumberOfTests(str) {
   return match && parseInt(match[1], 10);
 }
 
-var TOTAL_NUM_TESTS = 50;
+var TOTAL_NUM_TESTS = 53;
+
+function getTotalNumberOfTests(output) {
```

Comment:
What if we pass `{ setupEmberOnerrorValidation: false }` to `start()` in ember-exam/tests/test-helper.js ?

Reply:
This got marked as resolved, but I don't see a change.
But to reiterate:
What if we pass `{ setupEmberOnerrorValidation: false }` to `start()` in `ember-exam/tests/test-helper.js`, then we would not need this method to modify TOTAL_NUM_TESTS depending on the number of browsers.

Reply:
even if we pass { setupEmberOnerrorValidation: false } then getTotalNumberOfTests() will return 
 whatever number that's defined as TOTAL_NUM_TESTS.  getTotalNumberOfTests() parses test execution output file and subtract number of Ember.onError validation tests.